### PR TITLE
Replace bento by bootstrap for package(s)

### PR DESCRIPTION
**Pull request by saraycp**
Merge controllers, moving webui2_* actions to webui package(s) controller.
Replace package and packages bento views by bootstrap ones.
Adapt affected feature tests.

### CODE REVIEW

File: src/api/app/controllers/webui/package_controller.rb [hunk 1]
```diff
@@ -147,23 +146,14 @@ def dependency
   end
   # rubocop:enable Lint/NonLocalExitFromIterator
 
-  # TODO: bento_only
   def statistics
-    return if switch_to_webui2
```
Comment (DavidKang) [inline src/api/app/controllers/webui/package_controller.rb]: I think we still need this. not the `return` but the `switch_to_webui2`

Comment (saraycp) [reply src/api/app/controllers/webui/package_controller.rb]: Done!

File: src/api/app/controllers/webui/package_controller.rb [hunk 2]
```diff
@@ -150,23 +149,15 @@ def dependency
...
-    @arch = params[:arch]
```
[the remark this answers is not in the document]
Comment (krauselukas) [reply src/api/app/controllers/webui/package_controller.rb]: @dmarcoux I think this conversation is resolved right? I also checked again, and I think removing the ```@arch``` is fine :)

**Comment by krauselukas** [comment]
@dmarcoux @DavidKang Please review again, I think everything is addressed :)

**Comment by hennevogel** [comment]
coverage/coveralls hangs. coverage is fine, see [codecov](https://codecov.io/github/saraycp/open-build-service/commit/b1174fe2c8fd55aea0772181215dad91631118d8). merging.

**Pull request merged**